static/js/settings_org.js

```diff
@@ -3,11 +3,104 @@ const render_settings_admin_auth_methods_list = require('../templates/settings/a
 const render_settings_admin_realm_domains_list = require("../templates/settings/admin_realm_domains_list.hbs");
 const render_settings_admin_realm_dropdown_stream_list = require("../templates/settings/admin_realm_dropdown_stream_list.hbs");
 const render_settings_organization_settings_tip = require("../templates/settings/organization_settings_tip.hbs");
+const pygments_data = require("../generated/pygments_data.json");
 
 const meta = {
     loaded: false,
 };
 
+exports.default_code_language_widget = (function (element_id) {
```

Comment:
Is the list of languages so long that it makes sense to not just use a normal dropdown for the UI here?

Reply:
I guess the million JS variants argue for it.

I think ideally this widget would be done more generically, where we have a common function that accepts the `element_id` and list of options, with a name like `settings_ui.filterable_dropdown_menu`, and we just use it for this purpose; if it's not hard to do that refactor now, it'd save time testing such a refactor if/when we have another use case for it.

Reply:
Yes, it was troublesome to scroll through the list. Any kind of sorting wouldn't make sense either (we have around 30 languages we treat as higher priority, but that would just confuse people scrolling through the list).

Reply:
> I guess the million JS variants argue for it.

Hah, exactly. Also, since this was the first widget of this type, I didn't make it generic yet, but I want to push 1 commit here that merges at least these 3: the default language and the notifications stream widgets.

I felt that that was out of the scope for this particular PR, but I don't see why that would be a tough refactor. If you're fine with merging this as it is, I'd push a PR out tomorrow building upon this new widget's structure for all three.

Reply:
OK, wfm.

Reply:
And we should take advantage of the opportunity to just have CSS for the `.dropdown-search` class, rather than doing a rule for each widget with one (etc.).